### PR TITLE
Qumulo cli refresh

### PR DESCRIPTION
**Pull request by roytaishi**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by roytaishi** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by kairu-ms** [comment]
@necusjz please help check why the generated code cannot be loaded correctly in CI

File: src/qumulo/azext_qumulo/aaz/latest/qumulo/storage/file_system/_create.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
             arg_group="Properties",
             help="Marketplace details",
         )
-        _args_schema.private_i_ps = AAZListArg(
+        _args_schema.private_ips = AAZListArg(
             options=["--private-i-ps"],
```
Comment (necusjz) [inline src/qumulo/azext_qumulo/aaz/latest/qumulo/storage/file_system/_create.py:105]: pls update the option name as well

**Comment by necusjz** [comment]
> Hi @roytaishi
> 
> ## Release Suggestions
> ### Module: qumulo
> * Please log updates into to `src/qumulo/HISTORY.rst`
> * Update `VERSION` to `2.0.0` in `src/qumulo/setup.py`
> 
> #### Notes
> * For more info about extension versioning, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

ping on this

**Review by necusjz — APPROVED** [review]

**Pull request merged**